Header header    # standard ROS header, contains timestamp and coordinate information
float64 x      # position x
float64 y      # position y
float64 psi    # heading
float64 v      # velocity
float64 curvature  # curvature
float64 s        # s coordinate
float64 d        # d coordinate